### ISSUE TITLE
VTOL VR (667970)

### ISSUE THREAD

**Issue by E3FxGaming** [ValveSoftware/Proton#2916]
# Compatibility Report
- Name of the game with compatibility issues: VTOL VR
- Steam AppID of the game: 667970

## System Information
- GPU: GTX 1080 Ti
- Driver/LLVM version: nvidia 430.26
- Kernel version: 5.1.19-1
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/E3FxGaming/4371d57b063444a7c211ef20e75d0d21
- Proton version: 4.2-9

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-667970.log](https://github.com/ValveSoftware/Proton/files/3438632/steam-667970.log)

## Symptoms

Vive wands don't work - at all. They're detected in SteamVR Home, but don't show up at all in VTOL VR. Opening the SteamVR menu makes them visible and usable again, but closing the menu makes them disappear and be unusable again.

## Reproduction

Launch the game. Attempt to use controllers.

The controllers worked just fine earlier this year. I can't tell whether or not it has anything to do with the Index launch, but it looks like something screwed up the controller bindings (the game doesn't receive any controller input information anymore).

This report is very similar to #2836 (Symptoms/Reproduction-wise).

The fixes from this recent pull request may fix it #2882 .

**Comment by aeikum** [comment]
We updated our openvr support in Proton 4.11. Can you please retest with 4.11 and see if this is working now?

**Comment by E3FxGaming** [comment]
Hi, sorry for reporting late. Only got back to my VR setup yesterday.
Thank you very much for your effort, I'm able to play the game once more thanks to your Proton 4.11 efforts.

I played the game yesterday for half an hour but had to call it a day after that because of problems with my VR setup. Today I was able to fix my setup and played VTOL VR for an hour without any hickups or other things to complain about. HTC Vive wands (controllers) were tracked absolutely accurately - I couldn't be happier about the result.
